source/main_module/main.c

```diff
@@ -235,24 +242,24 @@ int main(void){
     {
         HardFault_Handler();
     }
+    PHAL_writeGPIO(SDC_CTRL_GPIO_Port, SDC_CTRL_Pin, 1);
 
 
     /* Task Creation */
     schedInit(APB1ClockRateHz);
     configureAnim(preflightAnimation, preflightChecks, 60, 750);
 
-    // taskCreate(coolingPeriodic, 200);
+    taskCreate(coolingPeriodic, 100);
     taskCreate(heartBeatLED, 500);
-    taskCreate(monitorSDCPeriodic, 15);
+    taskCreate(monitorSDCPeriodic, 20);
```

Comment:
maybe have the scan occur quickly, but reduce the rate the message is sent (sent per each full scan) maybe every 100-500ms or so